webui/react/src/components/UserSettings.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import Avatar from 'components/Avatar';
+
+import css from './UserSettings.module.scss';
+
+interface Props {
+  username: string;
+}
+
+const UserSettings: React.FC<Props> = ({ username }: Props) => {
+  return (
+    <div className={css.base}>
+      <div className={css.field}>
+        <span className={css.label}>Avatar</span>
+        <span className={css.value}>
+          <Avatar hideTooltip large name={username} />
+        </span>
+      </div>
+      <div className={css.field}>
+        <span className={css.label}>Username</span>
+        <span className={css.value}>{username}</span>
+      </div>
+    </div>
+  );
+};
+
+export default UserSettings;
```

Comment:
**suggestion:** What do you think about moving this and the module SCSS files over to `/src/hooks/useModal/useModalUserSettings/` just because it won't really be a shared component and really only used by the modal hook itself.